subprojects/core/src/test/groovy/org/gradle/api/tasks/bundling/TarTest.groovy

```diff
@@ -31,32 +31,32 @@ class TarTest extends AbstractArchiveTaskTest {
 
     def "default values"() {
         expect:
-        tar.compression == Compression.NONE
+        tar.compression.get() == Compression.NONE
         tar.archiveExtension.get() == 'tar'
     }
 
     def "compression determines default extension"() {
         when:
-        tar.compression = Compression.GZIP
+        tar.compression.set(Compression.GZIP)
```

Comment:
🤔 Also here, It should also work if we just keep it `compression = Compression.GZIP`, no?